src/backbone/backbone.c

```diff
@@ -94,9 +94,11 @@ static int backbone_cb_id(struct backbone_id *id)
 
 static bool backbone_cb_update(void)
 {
-	struct backbone_id new = { 0 }, *id;
+	/* struct backbone_id = {{ 0 }}, *id; */  /* correctly zero-initialized */
+	struct backbone_id new, *id;
```

Comment:
We should probably rename all variables named `new` to `new_id` or something.  Just to at least try and avoid clashing with reserved C++ keywords.